File: msgraph-mail/dotnet/Users/MailFolders/MessageActionFlag.cs
using Kiota.Abstractions.Serialization;
using System;
using System.Collections.Generic;
using System.Linq;
namespace Graphdotnetv4.Users.MailFolders {
    public class MessageActionFlag : IParsable<MessageActionFlag> {
        public IDictionary<string, Action<MessageActionFlag, IParseNode>> DeserializeFields => new Dictionary<string, Action<MessageActionFlag, IParseNode>> {
        };
        public void Serialize(ISerializationWriter writer) {
        }
    }
}

File: msgraph-mail/typescript/src/users/mailFolders/messageActionFlag.ts
import {SerializationWriter, ParseNode, Parsable} from '@microsoft/kiota-abstractions';

export class MessageActionFlag implements Parsable<MessageActionFlag> {
    public serialize (writer: SerializationWriter) : void {
    };
    public deserializeFields () : Map<string, (item: MessageActionFlag, node: ParseNode) => void> {
        return new Map<string, (item: MessageActionFlag, node: ParseNode) => void>([
        ]);
    };
}
